Octokit.Reactive/Clients/ObservableRepositoryCommitsClients.cs

```diff
@@ -66,12 +77,24 @@ public IObservable<GitHubCommit> GetAll(string owner, string name)
         /// <returns></returns>
         public IObservable<GitHubCommit> GetAll(string owner, string name, CommitRequest request)
         {
+            return GetAll(owner, name, request, ApiOptions.None);
+        }
+
+        /// <summary>
+        /// Gets all commits for a given repository
+        /// </summary>
+        /// <param name="owner">The owner of the repository</param>
+        /// <param name="name">The name of the repository</param>
+        /// <param name="request">Used to filter list of commits returned</param>
+        /// <param name="options">Options for changing the API response</param>
+        /// <returns></returns>
+        public IObservable<GitHubCommit> GetAll(string owner, string name, CommitRequest request, ApiOptions options)
+        {
             Ensure.ArgumentNotNullOrEmptyString(owner, "owner");
             Ensure.ArgumentNotNullOrEmptyString(name, "name");
             Ensure.ArgumentNotNull(request, "request");
 
```

Comment:
Should we be passing ApiOptions.None rather than null? I haven't been too involved in these changes to date...  What do other merged ApiOptions changes do in similar situation?